### DISCUSSION TITLE
confuse between `cargo loco` and `loco` CLI

### DISCUSSION THREAD

**Discussion by sidoqi** [loco-rs/loco#271] [Q&A]
Why we need to separate between `cargo loco` and `loco`? Why not just use `loco` as the canonical CLI, then wrap the `cargo loco` related command.

I believe this can improve the user experience. wdyt?

**Comment by kaplanelad** [comment]
can you describe the user flow as you see it?

**Reply by sidoqi** [reply]
As someone new to loco, I'm always lookin to the help command in the CLI. I'm not sure which CLI I should use, so ended up to try both. I can't naturally differentiate when to use both. So the switching context is tedious

**Reply by kaplanelad** [reply]
It should be simple. 
`loco new` for creating a new project

inside the project user `cargo` as a prefix, as you are working with Rust cargo `test/run,` you have another command called loco..

what are your suggestions? can you describe the flow?

**Reply by sidoqi** [reply]
Like I said before, i would love to only use `loco` command without cargo related syntax, so we can hide the details. 

Hence, we just need to memorize the loco command instead the combination of cargo syntax.

Not really a pain point, but more about UX consistency

**Reply by jondot** [reply]
@jurukode I used to have alias to help with that. but soon enough I realized I need cargo for many other things (cargo test, etc)
`cargo loco` means "i'm developing with loco in Rust" so that you remember you're in development mode.

There *is something* nice about a single `loco` to rule them all, but we'll have to keep thinking about how to do that. Wrapping `cargo` would probably be considered not nice by most Rust devs.

**Reply by sidoqi** [reply]
I think aliasing is good workaround, thanks for the suggestion
